doc/introduction.md

```diff
@@ -14,3 +14,98 @@ The essential concepts in RxJS which solve async event management are:
 - **Subject:** is the equivalent to an EventEmitter, and the only way of multicasting a value or event to multiple Observers.
 - **Schedulers:** are centralized dispatchers to control concurrency, allowing us to coordinate when computation happens on e.g. `setTimeout` or `requestAnimationFrame` or others.
+
+## First examples
+
+Normally you register event listeners.
+```js
+var button = document.querySelector('button');
+button.addEventListener('click', () => console.log('Clicked!'));
+```
+
+Using RxJS you create an observable instead.
+```js
+var button = document.querySelector('button');
+Rx.Observable.fromEvent(button, 'click')
+  .subscribe(() => console.log('Clicked!'));
+```
+
+
+### Purity
+What makes RxJS powerful is its ability to produce values using pure functions. That means your code is less prone for errors.
+
+Normally you would create an impure function, where other
+pieces of your code can mess up your state.
+```js
+var count = 0;
+var button = document.querySelector('button');
+button.addEventListener('click', () => console.log(`Clicked ${++count} times`));
+```
+
+Using RxJS you isolate the state.
+```Js
+var button = document.querySelector('button');
+Rx.Observable.fromEvent(button, 'click')
+  .scan(count => count + 1, 0)
+  .subscribe(count => console.log('Clicked ${count} times'));
+```
+
+The **scan** operator works just like **reduce** for arrays. It takes a value which is exposed to a callback. The returned value of the callback will then become the next value exposed the next time the callback runs.
+
+### Flow
+RxJS has a whole range of operators that helps you control how the events flow through your observables.
+
+This is how you would allow at most one click per second, with plain JavaScript:
+```js
+var count = 0;
+var lastClick = null;
+var button = document.querySelector('button');
+button.addEventListener('click', () => {
+  if (lastClick && Date.now() - lastClick >= 1000) {
+    console.log(`Clicked ${++count} times`);  
+  } else {
+    lastClick = Date.now();
+  }
+});
+```
+
+With RxJS:
+```js
+var button = document.querySelector('button');
+Rx.Observable.fromEvent(button, 'click')
+  .throttle(1000)
+  .scan(count => count + 1, 0)
+  .subscribe(count => console.log('Clicked ${count} times'));
+```
+
+Other flow control operators are [**filter**](/class/es6/Observable.js~Observable.html#instance-method-filter), [**delay**](/class/es6/Observable.js~Observable.html#instance-method-delay), [**debounceTime**](/class/es6/Observable.js~Observable.html#instance-method-debounceTime), [**take**](/class/es6/Observable.js~Observable.html#instance-method-take), [**takeUntil**](/class/es6/Observable.js~Observable.html#instance-method-takeUntil), [**distinct**](/class/es6/Observable.js~Observable.html#instance-method-distinct), [**distinctUntilChanged**](/class/es6/Observable.js~Observable.html#instance-method-distinctUntilChanged) etc.
```

Comment:
Great, thanks for the modifications @christianalfoni. I'm ready to give thumbs up for this PR, but there is one last detail to fix. These links are broken. All you need to do is change `/class/` to `../class/` and then that's it.